ChangeLog

```diff
@@ -1,3 +1,8 @@
+2015-08-17  Maximilian Hils <pyopenssl@maximilianhils.com>
+
+	* OpenSSL/SSL.py, : Add support for the ``MSG_PEEK`` flag to
```

Comment:
while we have shit terrible changlog format you’ll have to add `OpenSSL/test/test_ssl.py` here too (and no trailing comma please, it slid in the previous entry, feel free to remove it :))